### PR TITLE
Feature/hydra dvc poetry

### PR DESCRIPTION
**Pull request by krzysztofwolk-digica**
Implemented:

- Basic dvc config
- Poetry init

Readme is modified clariant copy.

### CODE REVIEW

**Review by igorzaton-digica — APPROVED** [review]
Nie mam uwag, nie znalazłem też żadnych błędów w README.

File: README.md
Comment (karolinawarycha-digica) [inline README.md:1]: this file includes a lot of generic starting tips which can be useful to use in every project but I'm not 100% sure if they should be in the README. the chances that sb will actually read them are higher but these are some guides about how to write the code. Digica Hub seems like an alternative.
If you decide to leave it here I would suggest to make use of a table of contents.

**Review by karolinawarycha-digica — CHANGES_REQUESTED** [review]

**Comment by krzysztofwolk-digica** [comment]
@krzysztofkrenc-digica jedziesz

File: README.md
Comment (krzysztofwolk-digica) [reply README.md:1]: Well, I do not have a strong opinion on that topic tbh, hub seems good as well. @JamieChojnacki  , @krzysztofkrenc-digica

Comment (krzysztofkrenc-digica) [reply README.md:1]: Then maybe it should be in a different repo? The goal here is to make DVC more accessible and I think this README is a "good start". What I'm missing here are some example dvc.yaml, params.yaml files that could be played with and changed to the user's liking, but should work out of the box. Another possibility is to make them run but raise something like DVCNotImplementedError. But at any rate, we need some small working example here and it won't work on DigicaHUB.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,94 @@
...
+* `static_files` - not executable files (if not needed may be removed)
+  * `general_configs` - configurations of application run (static tmp folder paths etc.)
+  * `hydra_configs` - hydra configuration files (model configs etc.)
+
+## DVC pipeline setup
+**Remember to provide description in your project**
+
+This is generic step-by-step approach of creating the pipeline. 
+
+To create pipeline You should:
```
Comment (krzysztofkrenc-digica) [inline README.md:37]:
```suggestion
To create pipeline you should:
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,94 @@
...
+- add necessary data to dvc with `dvc add`
+- create `.yaml` file containing all algorithm parameters, default setting: `params.yaml` file in the main directory
+- add new stage with `dvc stage add` or add and run new stage with `dvc run`
+- run all stages with `dvc repro`
+- set dvc config to proper local remote dir (`/project_name/dvc` try to keep naming convention) and use `dvc push` to store experiment on external storage
+- use `git commit` to store version of algorithm
```
[the remark this answers is not in the document]
Comment (krzysztofwolk-digica) [reply README.md:43]: Well, I need Instructions how to do it as well... Can you do it?

**Comment by krzysztofwolk-digica** [comment]
I tried to simplify the readme. Please review agian.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,71 @@
...
+General project description.
 
-## Author
+## Setup
+
+0. adjust `.toml` file **initial step only**
+1. install project required Python (recommended version ^3.9) 
+2. install poetry `pip install poetry`
+3. install poetry environment `poetry install`
+4. Adjust local_remote path in .dvc/config
```
Comment (karolinawarycha-digica) [inline README.md:10]: trochę nie rozumiem tych zawirowań z wielką i małą literą na początku, ale poza tym chyba ok :)

**Review by karolinawarycha-digica — APPROVED** [review]

**Review by krzysztofkrenc-digica — APPROVED** [review]
Please, add missing newlines.

**Pull request merged**